apps/TutorialServo/TutorialServo.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TUTORIAL_SERVO_H
+#define TUTORIAL_SERVO_H
+
+#include "mbed.h"
+
+
+class TutorialServo
+{
+  public:
+  TutorialServo(PinName servoPin, float servoFrequency, float degrees, float minPulseWidthFloat = 1, float maxPulseWidthFloat = 2);
+  
+  void setPositionInDegrees(float angleRange);
+  void setPulseWidthRangeInMs(float minPulsewidthMsFloat, float maxPulsewidthMsFloat);
```

Comment:
the same idea kinda applies to setPulseWidthRangeInMs(). this is only ever done once really. plus the info needed is already given in the constructor so maybe this should only be called in the constructor

Reply:
Fixed!